lib/Service/OpenProjectAPIService.php

```diff
@@ -453,6 +453,34 @@ public static function validateURL(string $url): bool {
 			preg_match('/^https?/', $url);
 	}
 
+	/**
+	 * validates the provided data for integration setup
+	 *
+	 * @param array<mixed> $values
+	 * @return bool
+	 * @throws \InvalidArgumentException
+	 *
+	 */
+	public static function validateIntegrationSetupInformation(array $values): bool {
+		foreach ($values as $key => $value) {
+			if ($key == 'openproject_instance_url' && !OpenProjectAPIService::validateURL((string)$value)) {
```

Comment:
always use `===` to check also the type
```suggestion
			if ($key === 'openproject_instance_url' && !OpenProjectAPIService::validateURL((string)$value)) {
```
why casting `$value` to string?
shouldn't `validateURL` return false if the parameter is not string?